### PR TITLE
Frog Ruins feature and edits to tree stump

### PR DESCRIPTION
**Pull request by Dweblenod**
Adds a new kind of ruins Feature slightly buried into the ground that generates in the normal and rough biomes of the Frog Land Type.

Includes changes to frequency of generation for tree stump

Structure created by Boxfox

### CODE REVIEW

**Comment by kirderf1** [comment]
This will cause a conflict with #557. We should decide which PR goes first, and which PR should adapt after the other one. @Dweblenod Would you be fine with resolving things on this PR after first merging #557, or would you rather see this PR merged first and leave me to fix the conflicts in #557?

**Comment by Dweblenod** [comment]
@kirderf1 I am good with https://github.com/lunar-sway/minestuck/pull/557 being merged first!

> This will cause a conflict with #557. We should decide which PR goes first, and which PR should adapt after the other one. @Dweblenod Would you be fine with resolving things on this PR after first merging #557, or would you rather see this PR merged first and leave me to fix the conflicts in #557?

**Comment by Dweblenod** [comment]
Changes from #557 now included!

**Review by kirderf1 — APPROVED** [review]

**Pull request merged**
